src/components/Calculator.js

```diff
@@ -0,0 +1,30 @@
+import '../styles/calculator.css';
+
+const Calculator = () => (
+  <div className="calculator">
+    <div className="calculator-display">0</div>
+    <div className="calculator-buttons">
+      <button className="calculator-button" type="button">AC</button>
+      <button className="calculator-button" type="button"> +/- </button>
+      <button className="calculator-button" type="button"> % </button>
+      <button className="calculator-button right" type="button"> ÷ </button>
+      <button className="calculator-button" type="button"> 7 </button>
+      <button className="calculator-button" type="button"> 8 </button>
+      <button className="calculator-button" type="button"> 9 </button>
+      <button className="calculator-button right" type="button"> x </button>
+      <button className="calculator-button" type="button"> 4 </button>
+      <button className="calculator-button" type="button"> 5 </button>
+      <button className="calculator-button" type="button"> 6 </button>
+      <button className="calculator-button right" type="button"> - </button>
+      <button className="calculator-button" type="button"> 1 </button>
+      <button className="calculator-button" type="button"> 2 </button>
+      <button className="calculator-button" type="button"> 3 </button>
+      <button className="calculator-button right" type="button"> + </button>
+      <button className="calculator-button" id="zero" type="button"> 0 </button>
```

Comment:
- [ ] Kindly style your calculator component a little bit moe so as to match the given design better. Adding a width property to the main container of the calculator and setting it to `30vw` achieves this as well as reducing the `grid-gap` to `1px` improves the design much more.

Before
![image](https://user-images.githubusercontent.com/81932478/220592690-5107746a-991f-4326-a921-b3189da1272f.png)


After adding further styling
![image](https://user-images.githubusercontent.com/81932478/220592593-50f731b9-47ce-4c2e-a22b-855def69e600.png)


Given design
![image](https://user-images.githubusercontent.com/81932478/220590962-828abb2a-8086-49cd-a5d7-2c489691388a.png)

Reply:
Thank you for the review and suggestions. Really helpful!